src/Http/Http.Extensions/src/HttpJsonServiceExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http.Json;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Http;
+
+/// <summary>
+/// Extension methods to configure JSON serialization behavior.
+/// </summary>
+public static class HttpJsonServiceExtensions
+{
+    /// <summary>
+    /// Configures options used for reading and writing JSON by route handlers.
+    /// </summary>
+    /// <remarks>
+    /// The options configured here will only affect JSON returned and processed
+    /// from route handlers, not controllers.
```

Comment:
This also applies to the ReadFromJson and SendAsJson methods right?

Reply:
Correct!